src/ui/units/dash/containers/FixedHeader/FixedHeader.scss

```diff
@@ -0,0 +1,108 @@
+.dash-fixed-header {
+    &__controls {
+        display: flex;
+        flex: 1;
+        padding: 8px 0;
+        min-height: 60px;
+
+        &_fixed {
+            position: fixed;
+            top: 40px;
+            background-color: var(
+                --dl-color-fixed-header-background,
+                var(--g-color-base-background)
+            );
+            padding-left: 24px;
+            padding-right: 24px;
+            margin-left: -24px;
+            z-index: 10;
+        }
+
+        &_edit-mode {
+            .react-grid-layout {
+                background-color: var(--g-color-base-background);
+            }
+        }
+    }
+
+    &__controls-placeholder {
+        height: 60px;
+        display: flex;
+        flex-direction: row;
+    }
+
+    &__controls-grid {
+        display: flex;
+        flex-grow: 1;
+
+        > .react-grid-layout {
+            flex: 1;
+            transition: none;
+        }
+    }
+
+    &__controls-settings {
+        width: (100% / 36) * 1;
+        flex-shrink: 0;
+        position: relative;
+    }
+
+    &__controls-settings-wrapper {
+        position: absolute;
+        inset: 0px 0px 0px 8px;
+        display: flex;
+        min-width: 28px;
+        align-items: center;
+        justify-content: end;
+    }
+
+    &__container-placeholder {
+        margin-bottom: 22px;
+    }
+
+    &__container-wrapper {
+        display: flex;
+        padding-bottom: 8px;
+        flex-direction: row;
+        flex: 1;
+
+        > .react-grid-layout {
+            flex: 1;
+            transition: none;
+        }
```

Comment:
It's better add one mixin for part
```
> .react-grid-layout {
            flex: 1;
            transition: none;
        }
```
and another for 
```
> .react-grid-layout {
            flex: 1;
            transition: none;
        }
```
Both mixins could be local or in common components style file